backend/fms_core/viewsets/taxon.py

```diff
@@ -32,7 +32,7 @@ def search(self, _request):
             query.add(Q(name=search_input), Q.OR)
         else:
             query = Q(id__icontains=search_input)
-            query.add(Q(name__startswith=search_input), Q.OR)
+            query.add(Q(name__icontains=search_input), Q.OR)
```

Comment:
I think the request was mostly to add a search by ncbi_id.

Reply:
oops, forgot to push that